homeassistant/components/tractive/__init__.py

```diff
@@ -129,6 +129,13 @@ async def _generate_trackables(
     if not trackable["device_id"]:
         return None
 
+    if "details" not in trackable:
+        _LOGGER.warning(
```

Comment:
Should this be just info?

As in, does a user to be alerted for this? And what can they do to resolve the issue in such a case?

```suggestion
        _LOGGER.info(
```

Reply:
Maybe indeed warning is not needed here. At the moment, the user cannot do much with this problem. I don't know if this is a permanent change in the Tractive API or just a bug. Robert and the issue reporter opened a ticket in Tractive support. Currently, the only workaround is to configure with HA all accounts to which trackers are assigned (this will be possible when we merge this change).